aee.js

```diff
@@ -72,3 +72,14 @@ Array.prototype.count = function (spec) {
   }
   return result
 }
+
+Array.prototype.index = function (spec) {
+  const len = this.length
+  if (len === 0 || spec === undefined) return -1
+  let searchSpec = (typeof spec === 'function') ? spec
```

Comment:
I plan to merge this as it is, and as soon as all the code is together, refactor this kind of things, as we now have unit tests! :) What do you think?